src/App.js

```diff
@@ -1,10 +1,37 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import TopMenu from "./components/TopMenu";
+import Pokemon from "./pages/Pokemon";
+import axios from "axios";
 
 function App() {
+    const [namePokemon, setNamePokemon] = useState([]);
```

Comment:
Zodra de setter aangeroepen wordt dan is het geen array mee. Ik zou dan de default waarde `undefined` maken
```suggestion
    const [namePokemon, setNamePokemon] = useState();
```